presto-main/src/main/java/com/facebook/presto/metadata/TableProperties.java

```diff
@@ -32,10 +31,10 @@
 
 public class TableProperties
 {
-    private final TableLayoutHandle handle;
+    private final TableHandle handle;
```

Comment:
Now this might not be so intuitive why `TableProperties` contains `TableHandle` at first glance. 

Do we need `TableHandle` here? Looks like we already have `ConnectorLayoutProperties`, so maybe we can just add `ConnectorId` and `ConnectorTransationalHandle`, similar to https://github.com/prestosql/presto/pull/363/commits/47ce7e62491691eb91126c68db09f5eca1e080ff#diff-f9111c4505177b8449a12f4cc556d3daR36 ? :)

Reply:
This commit is dropped.